docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -0,0 +1,140 @@
+
+After setting the `experimentalFeatures={{ clipboardPaste: true }}` prop, you can copy and paste data directly to and from your Data Grid.
+
+This new feature is extremely versatile, and you can exchange data from other fields, other grids, and even directly to and from the spreadsheet tool of your choice.
+
+<video preload="metadata" style="margin-bottom: 16px;" autoplay muted loop playsinline>
+  <source src="/static/blog/mui-x-mid-v6-features/copy_paste.mov" type="video/mp4">
+</video>
+
+This feature is integrated with the editing API, so pasted data can be persisted using the `processRowUpdate` prop to update your data source through your usual editing validation process.
+The callbacks `clipboardPasteStart` and `clipboardPasteEnd` are fired during the clipboard paste operation, which can be useful for extra customizations around the event.
+
+For more details on how to use clipboard copy and paste, check out the [Data Grid—Clipboard documentation](https://mui.com/x/react-data-grid/clipboard/).
+
+## Charts - alpha version
+
+We are delighted to announce the upcoming expansion of MUI X: a brand-new suite of components for building and customizing charts.
+
+With MUI X Charts, you can choose from a wide range of chart types, including line charts, bar charts, pie charts, area charts, scatter plots, and more.
+Each chart type is thoughtfully crafted with attention to detail, ensuring that the visual representations are not only aesthetically pleasing, but also highly effective in conveying complex data.
+
+And it ships with gorgeous palettes that were specially crafted by our designers!
+
+Check out the video below, highlighting some of our Charts:
+
+<video preload="metadata" style="margin-bottom: 16px;" autoplay muted loop playsinline>
+  <source src="/static/blog/mui-x-mid-v6-features/charts_final_v1.mov" type="video/mp4">
+</video>
+
+As we progress toward the stable version, we're committed to enhancing the overall experience for developers and users, so your feedback is fundamental!
+
+We will also steadily expand our portfolio with new chart types such as [Heat map](https://mui.com/x/react-charts/heat-map/), [Funnel](https://mui.com/x/react-charts/funnel/), [Gantt](https://mui.com/x/react-charts/gantt/), and more.
+If there's a specific chart visualization you'd like us to prioritize, we encourage you to upvote the respective [issue on GitHub](https://github.com/mui/mui-x/issues?q=is%3Aissue+is%3Aopen+label%3A%22component%3A+charts%22+label%3A%22waiting+for+%F0%9F%91%8D%22).
+Your input can directly influence our development schedule, so don't hesitate to let us know what matters most to you!
+
+[Get started with charts now!](https://mui.com/x/react-charts/)
+
+## Tree View is now part of the MUI X family
+
+[Tree View](https://mui.com/x/react-tree-view/) will be migrating from the [lab](https://mui.com/material-ui/about-the-lab/) to Alpha and is now part of MUI X!
```

Comment:
for the next PR, another thing to consider is to make it clearer that it can be already downloaded and tested (as we expect more feedback and feature requests).

Something along the lines.
Tree View has migrated from the lab to Alpha and is now part of MUI X! You can now download the pre-release package for testing.

Another suggestion is to ensure users that MIT features remain MIT.
But perhaps we can simply later link to the dedicated blog post.

Reply:
Ah right, I was confused about this. I jump too quickly to a conclusion by changing the version to v5.11.1. Fixed in: 9bec39b0e833890f595d4f9136c801a90ca3e234. 

I think the problem was:

> [Tree View](https://mui.com/x/react-tree-view/) will be migrating from the [lab](https://mui.com/material-ui/about-the-lab/) to Alpha and is now part of MUI X!

A component can't be part of MUI X and not be migrated at the same time.